tests/vcs/git/GitCommitBearTest.py

```diff
@@ -203,7 +217,7 @@ def test_body_checks(self):
 
     def test_different_path(self):
         no_git_dir = mkdtemp()
-        self.git_commit("A shortlog that is too long is not good for history")
+        self.git_commit("Very long shortlog never good for a project history")
```

Comment:
> IIUC "long" is actually a verb. A verb may not always be something that shows action. It could also be a description. :P

Your use of long seems to be an adjective here (from my minimal understanding of grammar).
An adjective describes a noun. A verb described an action, state, or occurrence. Here you're describing the shortlog which is a [noun](http://www.merriam-webster.com/dictionary/log) ?

[Long as a verb](http://www.oxfordlearnersdictionaries.com/definition/english/long_3): "to want something very much especially if it does not seem likely to happen soon"
Also see: [Long as an adjective](http://www.oxfordlearnersdictionaries.com/definition/english/long_1?q=long)